guides/it/laptop-security.md

```diff
@@ -75,10 +75,119 @@ Made Tech do not have a preferred/approved Anti-virus tool for Linux, but it is
+After=clamav-daemon.service syslog.target network.target
+
+[Service]
+Type=simple
+User=root
+ExecStartPre=/bin/bash -c "while [ ! -S /var/run/clamav/clamd.ctl ]; do sleep 1; done"
+ExecStart=/usr/sbin/clamonacc -F --config-file=/etc/clamav/clamd.conf --log=/var/log/clamav/clamonacc.log --move=/root/quarantine
+
+[Install]
+WantedBy=multi-user.target
+```
+
+The `ExecStartPre` script here ensures that `clamd` waits for ClamAv to finish loading virus definitions into memory and create the required socket before starting up (Otherwise it would error).
+
+You will likely need to increase the number of inotify watchers available from the default 65536.
+
+```
+echo fs.inotify.max_user_watches=1048576 | sudo tee -a /etc/sysctl.conf && sudo sysctl -p
+```
+
+Enable and start the service:
+
+```
+sudo systemctl enable clamonacc
+sudo systemctl start clamonacc
+```
+
+Check that clamonacc is running correctly by tailing logs:
+
+```
+sudo tail /var/log/clamav/clamonacc.log
+```
+
+You should see output like:
+
+```
+--------------------------------------
+ClamInotif: watching '/home' (and all sub-directories)
+ClamInotif: watching '/var/www' (and all sub-directories)
+```
+
+If you see:
+
+```
+--------------------------------------
+ClamInotif: watching '/home' (and all sub-directories)
+ClamInotif: watching '/var/www' (and all sub-directories)
+ERROR: ClamInotif: could not watch path '/home', No space left on device
+ClamScanQueue: stopped
+Clamonacc: stopped
+```
+
+That likely indicates you need to increase the number of inotify watchers available as mentioned above.
+
+We can test this is running correctly by downloading a test file:
+
+```
+$ wget www.eicar.org/download/eicar.com
+$ ls | grep eicar
+$ sudo ls /root/quarantine
+eicar.com
+```
+
+You'll see the file has not been downloaded to the directory you're in, but rather moved directly to the quarantine directory we set up earlier.
+
 The following resources are useful for configuring ClamAV:
 - The [Configuration](https://docs.clamav.net/manual/Usage/Configuration.html) section of the ClamAV documentation
 - The [On Access Scanning](https://docs.clamav.net/manual/OnAccess.html) section of the official docs
-- [Installation & Configuration of ClamAV Antivirus on Ubuntu 18.04](https://aaronbrighton.medium.com/installation-configuration-of-clamav-antivirus-on-ubuntu-18-04-a6416bab3b41) - this Medium article was 
+- [Installation & Configuration of ClamAV Antivirus on Ubuntu 18.04](https://aaronbrighton.medium.com/installation-configuration-of-clamav-antivirus-on-ubuntu-18-04-a6416bab3b41) - this Medium article was
 particularly helpful for installing on Ubuntu and enabling on-access scanning.
-
-The official Ubuntu PPA does not always provide the most recent version - at the time of writing the official PPA ships clamAV `0.103.2`, the most recent version is `0.103.3`.  Given that `0.103.3` was release more than 30 days ago, the PPA does not meet the requirement for running the latest version.
```

Comment:
I think we should leave this is following the conversation with Sam

Reply:
Was just about to amend it to include compilation instructions. Will split this out into a separate section at the same time.